src/helpers/HTTPTransport.ts

```diff
@@ -0,0 +1,79 @@
+const METHODS = {
+    GET: 'GET',
+    PUT: 'PUT',
+    POST: 'POST',
+    DELETE: 'DELETE'
+};
+
+interface OptionsType {
+    header?: string
+    data?: string
+    method?: string
+    timeout?: number
+}
+
+function queryStringify(data: string) {
+    const result = [];
+    for(let [key, value] of Object.entries(data)) {
+        result.push(`${key}=${value.toString()}`);
+    }
+    return '?' + result.join('&');
+}
```

Comment:
1) не очень понял, почему  data имеет тип string. Если я правильно понял, эта функция должны объект превращать в query-строку. Может лучше задать тип ```Record<string, string>```?
2) можно написать проще
```suggestion
function queryStringify(data: Record<string, string>) {
    return "?" + Object
      .entries(data)
      .map(([key, value]) => `${key}=${value}`)
      .join("&")
}
```